crates/playground/js/__tests__/Welcome.test.tsx

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from "@testing-library/react";
+import Welcome from "../Welcome";
```

Comment:
Maybe not right now, but in the future we could enable absolute imports in tsconfig/webpack config.